<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCL1140Class</class>
 <widget class="QMainWindow" name="PCL1140Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCL1140</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btn_frontView">
          <property name="text">
           <string>正视图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_topView">
          <property name="text">
           <string>俯视图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_leftView">
          <property name="text">
           <string>左视图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_pointSize">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_setColor">
          <property name="text">
           <string>设置点云颜色</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QVTKOpenGLNativeWidget" name="openGLWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu234">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen_pcd"/>
    <addaction name="actionOpen_las"/>
   </widget>
   <addaction name="menu234"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_pcd">
   <property name="text">
    <string>打开pcd</string>
   </property>
  </action>
  <action name="actionOpen_las">
   <property name="text">
    <string>打开las</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>qvtkopenglnativewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PCL1140.qrc"/>
 </resources>
 <connections/>
</ui>
